packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -169,10 +171,14 @@ describe("wrangler secret", () => {
         mockAccountId({ accountId: null });
 
         it("should error if a user has no account", async () => {
+          mockGetMemberships({
```

Comment:
I don't think this will ever happen. If a user has no account, then the api will throw a 10000 error.

Reply:
You can reproduce this locally with a garbage invalid token.

Reply:
This was a preexisting test, I just updated to using a new mock for consistency.

Reply:
The existsing test had the right error message tho, this isn't the right mock. It will never return a list with zero memberships.

Reply:
Oh hmm I see. Aight never mind then.